src/assets/css/footer-section.css

```diff
@@ -0,0 +1,24 @@
+.LogoDarkStyle{
+    height: 100px;
+    margin-top: 24px;
+}
```

Comment:
Pls use hypen delimited strings example `.logo-dark`, please apply to other classes also